index.js

```diff
@@ -23,18 +23,17 @@ const binWrapper = (binInstallURL = null, binInstallBase = null) => {
     sources = sources.filter(x => process.platform === x.os && process.arch === x.arch);
 
     if (binInstallURL) {
-        bw.src(binInstallURL, '', '')
+        bw.src(binInstallURL, process.platform, process.arch);
```

Comment:
This won't cause any problems (new differences) with folks that already override the URL today? 😅

Reply:
That should have no impacts at all. I've tried it and it went without any issues (both with and without testing).

This change is due to a slight change in the logic of resolution between a source and platform/arch.
While before an empty platform/arch was considered as a catch-all value, now a value is enforced.

In both case, we specify a source for a the *current* platform/arch, so nothing should have any impact.
If you don't feel super confident on this, I can increase the testing spectrum.